addons/mail/models/mail_message_translation.py

```diff
@@ -0,0 +1,40 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+from dateutil.relativedelta import relativedelta
+
+
+class MessageTranslation(models.Model):
+    _name = "mail.message.translation"
+    _description = "Message Translation"
+
+    create_date = fields.Datetime(index=True)
+    message_id = fields.Many2one(
+        "mail.message",
+        "Message",
+        required=True,
+        ondelete="cascade",
+        index=True,
```

Comment:
not necessary to index message alone, as it is already part of the unique index as the first key